src/Symfony/Component/PropertyInfo/Tests/Fixtures/Php74Dummy.php

```diff
@@ -23,11 +23,18 @@ class Php74Dummy
     private ?int $nullableWithDefault = 1;
     public array $collection = [];
 
+    /** @var Dummy[]|null  */
```

Comment:
Does the `PhpDocExtractor` handle the described use case properly? Do we have a test covering it or would it be worth to add one?

Reply:
PhpDocExtractor works correctly for this use case, I've added tests covering that specifically